src/librustc_mir/borrow_check/borrow_set.rs

```diff
@@ -304,7 +304,6 @@ impl<'a, 'gcx, 'tcx> GatherBorrows<'a, 'gcx, 'tcx> {
     /// Activation phases.
     fn allow_two_phase_borrow(&self, kind: mir::BorrowKind) -> bool {
```

Comment:
Could you remove this function and replace uses of it with the one in `path_utils`?

Reply:
@matthewjasper Thanks for reviewing! I removed the function.